src/components/CallView/CallView.vue

```diff
@@ -78,11 +80,21 @@
 						:is-one-to-one="isOneToOne"
 						:is-sidebar="isSidebar"
 						@force-promote-video="forcePromotedModel = $event" />
+
+					<div v-else-if="devMode && !isGrid"
+						class="dev-mode-video--promoted">
+						<img :alt="placeholderName(6)" :src="placeholderImage(6)">
+						<VideoBottomBar :has-shadow="false"
+							:model="placeholderModel(6)"
+							:shared-data="placeholderSharedData(6)"
+							:token="token"
+							:participant-name="placeholderName(6)" />
+					</div>
```

Comment:
Bottom bar is not well placed
![image](https://github.com/user-attachments/assets/a1d78b8f-995e-40d6-adab-be41e4f0bd19)

Reply:
`is-big` prop was missing